### PR TITLE
feat(45145): add more links to developers page

### LINKED ISSUE #45145: Docs: Developers page could include more information and links to resources
The [Developers](https://grafana.com/docs/grafana/latest/developers/) page is currently quite empty , only containing 

> This section of the documentation contains pages with resources for Grafana developers.

Based on feedback, could be a good place to link to resources, like Grafana contribue guide, Tutorials, community site etc.

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What this PR does / why we need it**:
Adding the most valuable links to the developers root page in our documentation.

**Which issue(s) this PR fixes**:

Fixes #45145

**Special notes for your reviewer**:

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

**Comment by torkelo** [comment]
@tolzhabayev you need to backport doc site changes to latest stable for them to become live

File: docs/sources/developers/_index.md [hunk 1]
```diff
@@ -6,4 +6,10 @@ weight = 190
 
 # Developers
 
-This section of the documentation contains pages with resources for Grafana developers.
+This section of the documentation contains pages with resources for Grafana developers:
+
+- [Grafana Contribution Guide - learn how you can contribute your skills to make Grafana even better.](contribute)
+- [How to build a plugin](plugins)
```
Comment (marefr) [inline docs/sources/developers/_index.md:12]: Please use relref format for internal links, https://github.com/grafana/grafana/tree/main/docs#using-relref-for-internal-links

**Review by marefr — CHANGES_REQUESTED** [review]

File: docs/sources/developers/_index.md (hunk 1 above)
Comment (tolzhabayev) [reply docs/sources/developers/_index.md]: @marefr done

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**